server/src/test/java/com/hedera/block/server/config/ServerMappedConfigSourceInitializerTest.java

```diff
@@ -68,6 +68,8 @@ class ServerMappedConfigSourceInitializerTest {
 
         // Server Config
         new ConfigMapping("server.maxMessageSizeBytes", "SERVER_MAX_MESSAGE_SIZE_BYTES"),
+        new ConfigMapping("server.socketSendBufferSizeBytes", "SOCKET_SEND_BUFFER_SIZE_BYTES"),
+        new ConfigMapping("server.socketReceiveBufferSizeBytes", "SOCKET_RECEIVE_BUFFER_SIZE_BYTES"),
```

Comment:
Let's keep the convention and use the full key of the config:

```suggestion
        new ConfigMapping("server.socketSendBufferSizeBytes", "SERVER_SOCKET_SEND_BUFFER_SIZE_BYTES"),
        new ConfigMapping("server.socketReceiveBufferSizeBytes", "SERVER_SOCKET_RECEIVE_BUFFER_SIZE_BYTES"),
```